src/features/SendAssets/__tests__/SendAssets.reducer.test.tsx

```diff
@@ -0,0 +1,211 @@
+import { bigNumberify } from 'ethers/utils';
+
+import { IFormikFields, ITxStatus } from '@types';
+import { fAssets } from '@../jest_config/__fixtures__/assets';
+import { fAccount, fNetwork } from '@fixtures';
+import { getDefaultEstimates, inputGasPriceToHex } from '@services';
+
+import { sendAssetsReducer, ReducerAction } from '../SendAssets.reducer';
+
+const dispatch = (action: ReducerAction) => (state: any) => sendAssetsReducer(state, action);
+
+const defaultTxConfig = {
+  asset: fAssets[0],
+  baseAsset: fAssets[0],
+  amount: '1',
+  gasPrice: '57000000000',
+  gasLimit: '21000',
+  nonce: '10',
+  data: '0x',
+  rawTransaction: {
+    value: '1',
+    to: fAccount.address,
+    gasPrice: bigNumberify('57000000000')
+  },
+  from: fAccount.address,
+  senderAccount: fAccount,
+  receiverAddress: fAccount.address
+};
+
+jest.mock('ethers/utils', () => {
+  const { mockFactory } = require('../__mocks__/utils');
+  return mockFactory({
+    gasPrice: '57000000000',
+    gasLimit: '21000',
+    nonce: '10',
+    data: '0x',
+    receiverAddress: '0xfE5443FaC29fA621cFc33D41D1927fd0f5E0bB7c',
+    value: '10000000000000',
+    to: '0xfE5443FaC29fA621cFc33D41D1927fd0f5E0bB7c'
```

Comment:
Is this suppose to be the same as `defaultTxConfig`? Can we make it explicit?

Reply:
It's tricky with Jest since you can't use anything out the scope of the mock function.

Reply:
oh!